herald_rules/add_comment_for_workflows.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "This is a rule to attach @gagoar when workflow changes",
-  "includes": [".github/workflows/*"],
+  "includes": ["src/*"],
   "action": "comment",
-  "users": ["gagoar"]
+  "users": ["gagoar", "cyamonide", "jnwng"]
 }
```

Comment:
Was this meant as a test change to have herald post on the PR?
If so, don't forget to change it back. Otherwise, you should probably rename the rule to something more accurate.

Reply:
yea, this is just for seeing the change